### PR TITLE
Translate

### PR DESCRIPTION
**Pull request by Asa-Nguyen**

### CODE REVIEW

File: resources/views/auth/register.blade.php [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
                     <label class="form-label" for="name">{{ __('Name') }} <span class="text-danger">*</span></label>
                     <input type="text" class="form-control form-control-lg bg-white bg-opacity-5"
                            placeholder="{{ __('e.g John Smith') }}" name="name" id="name" value="{{ old('name') }}"/>
                 </div>
                 <div class="mb-3">
-                    <label class="form-label" for="email">{{ __('Email Address ') }}<span class="text-danger">*</span></label>
+                    <label class="form-label" for="email">{{ __('Email Address') }}<span class="text-danger">*</span></label>
                     <input type="text" class="form-control form-control-lg bg-white bg-opacity-5"
-                           placeholder="username@address.com" name="email" id="email" value="{{ old('email')}}"/>
+                           placeholder="{{ __('username@address.com') }}" name="email" id="email" value="{{ old('email')}}"/>
```
Comment (ntd1683) [inline resources/views/auth/register.blade.php]: email thì dịch làm gì huynh

File: resources/views/components/layouts/partials/header.blade.php [hunk 2]
```diff
@@ -51,27 +51,27 @@
                                 <i class="bi bi-circle-fill position-absolute text-theme top-0 mt-n2 me-n2 fs-6px d-block text-center w-100"></i>
                                 <i class="bi bi-envelope h2 opacity-5 d-block my-1"></i>
                             </div>
-                            <div class="fw-500 fs-10px text-white">INBOX</div>
+                            <div class="fw-500 fs-10px text-white">{{ __('INBOX')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                            <div class="fw-500 fs-10px text-white">{{ __('INBOX') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 3]
```diff
@@ -51,27 +51,27 @@
...
                         </a>
                     </div>
                     <div class="col-4">
                         <a href="#" class="dropdown-item text-decoration-none p-3 bg-none">
                             <div><i class="bi bi-hdd-network h2 opacity-5 d-block my-1"></i></div>
-                            <div class="fw-500 fs-10px text-white">DISK DRIVE</div>
+                            <div class="fw-500 fs-10px text-white">{{ __('DISK DRIVE')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                            <div class="fw-500 fs-10px text-white">{{ __('DISK DRIVE') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 4]
```diff
@@ -51,27 +51,27 @@
...
                         </a>
                     </div>
                     <div class="col-4">
                         <a href="#" class="dropdown-item text-decoration-none p-3 bg-none">
                             <div><i class="bi bi-calendar4 h2 opacity-5 d-block my-1"></i></div>
-                            <div class="fw-500 fs-10px text-white">CALENDAR</div>
+                            <div class="fw-500 fs-10px text-white">{{ __('CALENDAR')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                            <div class="fw-500 fs-10px text-white">{{ __('CALENDAR') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 5]
```diff
@@ -51,27 +51,27 @@
...
                         </a>
                     </div>
                 </div>
                 <div class="row row-grid gx-0">
                     <div class="col-4">
                         <a href="#" class="dropdown-item text-decoration-none p-3 bg-none">
                             <div><i class="bi bi-terminal h2 opacity-5 d-block my-1"></i></div>
-                            <div class="fw-500 fs-10px text-white">TERMINAL</div>
+                            <div class="fw-500 fs-10px text-white">{{ __('TERMINAL')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                            <div class="fw-500 fs-10px text-white">{{ __('TERMINAL') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 6]
```diff
@@ -80,13 +80,13 @@
                                 <i class="bi bi-circle-fill position-absolute text-theme top-0 mt-n2 me-n2 fs-6px d-block text-center w-100"></i>
                                 <i class="bi bi-sliders h2 opacity-5 d-block my-1"></i>
                             </div>
-                            <div class="fw-500 fs-10px text-white">SETTINGS</div>
+                            <div class="fw-500 fs-10px text-white">{{ __('SETTINGS')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                            <div class="fw-500 fs-10px text-white">{{ __('SETTINGS') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 7]
```diff
@@ -80,13 +80,13 @@
...
                         </a>
                     </div>
                     <div class="col-4">
                         <a href="#" class="dropdown-item text-decoration-none p-3 bg-none">
                             <div><i class="bi bi-collection-play h2 opacity-5 d-block my-1"></i></div>
-                            <div class="fw-500 fs-10px text-white">LIBRARY</div>
+                            <div class="fw-500 fs-10px text-white">{{ __('LIBRARY')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                            <div class="fw-500 fs-10px text-white">{{ __('LIBRARY') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 8]
```diff
@@ -98,14 +98,11 @@
...
-                <div
-                    class="d-flex align-items-center py-10px dropdown-item text-wrap fs-6 bg-white bg-opacity-15 text-white text-opacity-50 h-100px">
-                    No record found
-                </div>
+                <h6 class="dropdown-header fs-10px mb-1">{{ __('NOTIFICATIONS')}}</h6>
+                <div class="d-flex align-items-center py-10px dropdown-item text-wrap fs-6 bg-white bg-opacity-15 text-white text-opacity-50 h-100px">{{ __('No record found')}}</div>
                 <hr class="bg-white-transparent-5 mb-0 mt-0"/>
                 <div class="py-10px mb-n2 text-center">
-                    <a href="#" class="text-decoration-none fw-bold">SEE ALL</a>
+                    <a href="#" class="text-decoration-none fw-bold">{{ __('SEE ALL')}}</a>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                    <a href="#" class="text-decoration-none fw-bold">{{ __('SEE ALL') }}</a>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 9]
```diff
@@ -98,14 +98,11 @@
                 <div class="menu-badge bg-theme"></div>
             </a>
             <div class="dropdown-menu dropdown-menu-end mt-1 w-300px fs-11px pt-1">
-                <h6 class="dropdown-header fs-10px mb-1">NOTIFICATIONS</h6>
-                <div
-                    class="d-flex align-items-center py-10px dropdown-item text-wrap fs-6 bg-white bg-opacity-15 text-white text-opacity-50 h-100px">
-                    No record found
-                </div>
+                <h6 class="dropdown-header fs-10px mb-1">{{ __('NOTIFICATIONS')}}</h6>
+                <div class="d-flex align-items-center py-10px dropdown-item text-wrap fs-6 bg-white bg-opacity-15 text-white text-opacity-50 h-100px">{{ __('No record found')}}</div>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php:102]:
```suggestion
                <div class="d-flex align-items-center py-10px dropdown-item text-wrap fs-6 bg-white bg-opacity-15 text-white text-opacity-50 h-100px">{{ __('No record found') }}</div>
```

File: resources/views/components/layouts/partials/header.blade.php [hunk 10]
```diff
@@ -98,14 +98,11 @@
                 <div class="menu-badge bg-theme"></div>
             </a>
             <div class="dropdown-menu dropdown-menu-end mt-1 w-300px fs-11px pt-1">
-                <h6 class="dropdown-header fs-10px mb-1">NOTIFICATIONS</h6>
-                <div
-                    class="d-flex align-items-center py-10px dropdown-item text-wrap fs-6 bg-white bg-opacity-15 text-white text-opacity-50 h-100px">
-                    No record found
-                </div>
+                <h6 class="dropdown-header fs-10px mb-1">{{ __('NOTIFICATIONS')}}</h6>
```
Comment (ntd1683) [inline resources/views/components/layouts/partials/header.blade.php]:
```suggestion
                <h6 class="dropdown-header fs-10px mb-1">{{ __('NOTIFICATIONS') }}</h6>
```

File: resources/views/auth/register.blade.php (hunk 1 above)
Comment (Asa-Nguyen) [reply resources/views/auth/register.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 2 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 3 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 4 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 5 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 6 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 7 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 9 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php:102]: checked

File: resources/views/components/layouts/partials/header.blade.php (hunk 10 above)
Comment (Asa-Nguyen) [reply resources/views/components/layouts/partials/header.blade.php]: checked

**Review by ntd1683 — APPROVED** [review]

**Pull request merged**